functions.txt

```diff
@@ -516,6 +516,22 @@ define('PHP_URL_FRAGMENT', 7);
 function parse_url ($str ::: string, $component ::: int = -1) ::: mixed;
 
 define('M_PI', 3.1415926535897932384626433832795);
+define('M_E', 2.7182818284590452354);
+define('M_LOG2E', 1.4426950408889634074);
+define('M_LOG10E', 0.43429448190325182765);
+define('M_LN2', 0.69314718055994530942);
+define('M_LN10', 2.30258509299404568402);
+define('M_PI_2', 1.57079632679489661923);
+define('M_PI_4', 0.78539816339744830962);
+define('M_1_PI', 0.31830988618379067154);
+define('M_2_PI', 0.63661977236758134308);
+define('M_SQRTPI', 1.77245385090551602729);
+define('M_2_SQRTPI', 1.12837916709551257390);
+define('M_SQRT2', 1.41421356237309504880);
+define('M_SQRT3', 1.73205080756887729352);
+define('M_SQRT1_2', 0.70710678118654752440);
+define('M_LNPI', 1.14472988584940017414);
+define('M_EULER', 0.57721566490153286061);
 
```

Comment:
Please, add the `is_infinite()` function (it exists in PHP, like `is_nan()`)

Reply:
This function is already implemented here: functions.txt:307. Also it used in new test tests/phpt/dl/1029_inf.php.